build/build_app_engine_package.py

```diff
@@ -80,8 +80,7 @@ def main(src_path, dest_path):
           shutil.copy(os.path.join(dirpath, name), dest_html_path)
     elif dirpath.endswith('app_engine'):
       for name in files:
-        if (name.endswith('.py') and name.find('test') == -1 or
```

Comment:
It seemed hackish to me to just copy the tests when they're about to run. This will always copy them. Any reason why this would be a problem?